source/nodejs/ac-typed-schema/src/markdown/generate-markdown.ts

```diff
@@ -1,15 +1,46 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+import { I18n } from "i18n";
 import { defined } from "./defined";
 import { defaultValue } from "./defaultValue";
 import { SchemaClass } from "../SchemaClass";
 import { SchemaEnum } from "../SchemaEnum";
 import { SchemaLiteral } from "../SchemaLiteral";
 import { SchemaProperty } from "../SchemaProperty";
 import { SchemaPropertyType } from "../SchemaPropertyType";
+import * as path from "path";
+import * as console from "console";
 import * as mdTable from "markdown-table";
 import * as style from "./style";
 
+class MarkdownConfig {
+    i18n : I18n;
```

Comment:
Should this be `private`?

Reply:
need access to allow translation to be called from `adaptivecards-site` -- it's probably worth noting that this is a private module that's only used during generation of site content. :)

Reply:
perhaps private with a public getter would be more appropriate :)